week-1/sql-exercises.md

```diff
@@ -1,6 +1,6 @@
 # Round 1
 
-In these exercises we focus solely on writing SQL. Use [the online SQLite viewer](https://inloop.github.io/sqlite-viewer/), upload [our sample database file](chinook.sqlite) and start writing those queries!
+In these exercises we focus solely on writing SQL. Use [the online SQLite viewer](https://inloop.github.io/sqlite-viewer/), upload [our sample database file](/chinook-database) and start writing those queries!
```

Comment:
@Senait-coding , a full relative path would be more robust.  using just `/` works in some cases but won't always work depending on how the content is hosted, `../` should always work no matter what

Reply:
@evan , I've changed it  , hope it is okay now :) thanks